cpp/oneapi/dal/algo/covariance/detail/compute_ops.hpp

```diff
@@ -24,6 +24,13 @@ namespace v1 {
 
 template <typename Context, typename Float, typename Method, typename Task, typename... Options>
 struct compute_ops_dispatcher {
+    compute_result<Task> operator()(const Context&,
```

Comment:
Now here it makes obviously sense to use `Context& ctx`. But now I'm confused that a `Policy&` can be a `ctx`, too.

Reply:
I understand your concerns. For now I am just aligning with the rest of the code base. But yes, we'll need to define more clear rules regarding 'ctx' and 'policy' naming - where to use what.

Reply:
Yep, definitely something to keep in mind!